Library/Formula/geoipupdate.rb

```diff
@@ -25,7 +25,7 @@ def install
     # See https://github.com/maxmind/geoip-api-c#150
     inreplace "conf/GeoIP.conf.default", "YOUR_USER_ID_HERE", "999999"
     inreplace "conf/GeoIP.conf.default", "YOUR_LICENSE_KEY_HERE", "000000000000"
-    inreplace "conf/GeoIP.conf.default", /^ProductIds .*$/, "ProductIds 506 533"
+    inreplace "conf/GeoIP.conf.default", /^ProductIds .*$/, "ProductIds 506 533 GeoLite2-City GeoLite2-Country"
```

Comment:
Sorry, I should have made the commit or PR message more specific. They are licensed under the same license as GeoLite Legacy, CC BY-SA 3.0. With this configuration, they will be installed or updated when geoipupdate is run, as will the legacy databases.